pkgs/development/compilers/ispc/default.nix

```diff
@@ -7,15 +7,27 @@
 
 stdenv.mkDerivation rec {
   pname   = "ispc";
-  version = "1.13.0";
+  version = "1.15.0.2021-04-02";
 
   src = fetchFromGitHub {
     owner  = pname;
     repo   = pname;
-    rev    = "v${version}";
-    sha256 = "1l74xkpwwxc38k2ngg7mpvswziiy91yxslgfad6688hh1n5jvayd";
+    # ISPC release 1.15.0 doesn't build against LLVM 11.1, only against 11.0. So we
```

Comment:
Can we fetch patches instead?

Reply:
I guess we could, but I don't have the time for it. Just wanted to update this to bump `openimagedenoise` tbh.